raster/r.horizon/main.c

```diff
@@ -393,7 +393,7 @@ int main(int argc, char *argv[])
 	sscanf(parm.direction->answer, "%lf", &single_direction);
 
 
-    if (isMode(WHOLE_RASTER)) {
+    if (WHOLE_RASTER == isMode()) {
```

Comment:
`int  isMode()` in L132 should be changed to `int  isMode(void)`

Reply:
updated